### ISSUE TITLE
Fail to verify https-only site

### ISSUE THREAD

**Issue by muchweb** [keybase/keybase-issues#1449]
I am running `https`-only site. When accessed by `http`, server returns correct `HTTP/1.1 301 Moved Permanently`.

When I am trying to verify my site, I am getting following error:

>Could not query [*site name*]. Are you running a site there?

This is a mystery for me. In the dialogue, there is no was to implicitly specify a protocol.

**Comment by malgorithms** [comment]
what domain is it, and what's the path to the proof?

It must be something other than the 301 redirect that's causing a prob - that's pretty standard.

**Comment by muchweb** [comment]
>It must be something other than the 301 redirect that's causing a prob - that's pretty standard.

@malgorithms, yes, you are right. Thank you. I have fixed it.

I was missing one of certificates in the chain that is normally supported by the browsers, but I have tried it in plain server and `curl` was throwing an error. This tool have helped me: https://www.ssllabs.com/ssltest

It would still be nice to add more informative error message on the website.
